src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
@richardeakin: Sorry, for not answering earlier. I read the post, but then it got swept under the ruggs during the holidays. 

In any case, the implementation sounds good to me, but you probably also have to remove the copy constructor and copy assignment operator or specify the move constructor and move assignment operator as noexcept (at least on platforms other than VS2013). Otherwise, a c++11 conforming implementation of `std::vector` will use the copy assignment/constructor instead of the move operators when it moves things around (e.g. when using `push_back`). I believe VS2013 and VS2015 will still use the move operations , but e.g. gcc on linux will use the copy operators.